prototype_source/inductor_with_x86_cpu_as_backend_of_pt2e.rst

```diff
@@ -0,0 +1,186 @@
+TorchInductor with X86 CPU as backend of Quantization in PyTorch 2.0 Export
+=============================================================================
+
+**Author**: `Leslie Fang <https://github.com/leslie-fang-intel>`_, `Weiwen Xia <https://github.com/Xia-Weiwen>`_, `Jiong Gong <https://github.com/jgong5>`_, `Jerry Zhang <https://github.com/jerryzh168>`_
+
+Prerequisites
+^^^^^^^^^^^^^^^
+
+-  `PyTorch 2.0 Export Post Training Static Quantization <https://pytorch.org/tutorials/prototype/pt2e_quant_ptq_static.html>`_
+-  `TorchInductor and torch.compile concepts in PyTorch <https://pytorch.org/tutorials/intermediate/torch_compile_tutorial.html>`_
+
+Introduction
+^^^^^^^^^^^^^^
+
+This tutorial introduces the steps to use PyTorch 2.0 Export Quantization flow and TorchInductor to do post-training static quantization
+in graph mode on X86 CPU devices.
```

Comment:
nit: we can remove "graph mode", since pytorch 2.0 export is "mode" as well

Reply:
Thanks and removed.